data/filedefs/filetypes.asciidoc

```diff
@@ -1,35 +1,33 @@
 # For complete documentation of this file, please see Geany's main documentation
 [styling]
-# no syntax highlighting yet
+# Edit these in the colorscheme .conf file instead
+default=default
+strong=default,bold
+emphasis=default,italic
+header1=keyword_1
+header2=keyword_1
+header3=keyword_1
+header4=keyword_1
+header5=keyword_1
+header6=keyword_1
+ulist_item=default
+olist_item=default
+blockquote=default,italic
```

Comment:
Markdown uses tag_unknown for the above 3. Since markdown is similar in spirit it might make sense to align to that.

Reply:
Better than just default.

Reply:
The scheme was configured to match appearance of AsciiDoc extension in Visual Studio Code:

https://marketplace.visualstudio.com/items?itemName=asciidoctor.asciidoctor-vscode

Tomatch also colors, I use this local "filetypes.asciidoc" configuration file:

```
[styling]
strong=0x001080;0xffffff;true;false
header1=0x800000;0xffffff;true;false
header2=0x800000;0xffffff;true;false
header3=0x800000;0xffffff;true;false
header4=0x800000;0xffffff;true;false
header5=0x800000;0xffffff;true;false
header6=0x800000;0xffffff;true;false
link=0xa31515;0xffffff;false;false
comment=0x008000;0xffffff;false;false
attribval=0xa31515;0xffffff;false;false
macro=0x795e26;0xffffff;false;false
```

Reply:
Well, which colours are mapped to the named styles is the job of the Geany colour scheme, not the filetype file.  The filetype file should map lexical elements to named styles only, not to colours. 

To match VScode there should be a VScode colour scheme that will make all languages match (within the limits of the respective lexers).  But if you just map a lexical element to "default" its not going to be coloured at all.

Reply:
Probably the "right" solution would be to define some more named styles suitable for document markup languages (heading1-6, unordered_list, ordered_list, etc) instead of the typical Geany habit of trying to make all languages C ;-P

Then asciidoc, markdown, and rest (if it gets a lexer) can all use them.

Probably should have been done when markdown (the first document language lexer) was added.

Reply:
@ancebfer you're free to match VScode style on your machine. For this PR, please align to markdown as I suggested.

Reply:
Done